packages/edit-site/src/components/sidebar-navigation-screen-global-styles/index.js

```diff
@@ -25,16 +39,32 @@ export default function SidebarNavigationScreenGlobalStyles() {
 			) }
 			content={ <StyleVariationsContainer /> }
 			actions={
-				<SidebarButton
-					icon={ edit }
-					label={ __( 'Edit styles' ) }
-					onClick={ () => {
-						// switch to edit mode.
-						setCanvasMode( 'edit' );
-						// open global styles sidebar.
-						openGeneralSidebar( 'edit-site/global-styles' );
-					} }
-				/>
+				<div>
+					<SidebarButton
+						icon={ seen }
+						label={ __( 'Style Book' ) }
+						onClick={ () => {
+							// Switch to edit mode.
+							setCanvasMode( 'edit' );
+							// Open global styles sidebar.
+							openGeneralSidebar( 'edit-site/global-styles' );
+							// Open style book, via the useEffect above.
+							// This is done via a local state change to ensure that the canvas mode is changed,
+							// and the global styles sidebar is opened before attempting to open the style book.
+							setIsStyleBookOpened( true );
+						} }
+					/>
+					<SidebarButton
+						icon={ edit }
+						label={ __( 'Edit styles' ) }
+						onClick={ () => {
+							// Switch to edit mode.
+							setCanvasMode( 'edit' );
+							// Open global styles sidebar.
+							openGeneralSidebar( 'edit-site/global-styles' );
```

Comment:
Since we're reusing this, you could chain these up:

```js
	const openGlobalStyles = async () =>
		Promise.all( [
			setCanvasMode( 'edit' ),
			openGeneralSidebar( 'edit-site/global-styles' ),
		] );

...

					<SidebarButton
						icon={ edit }
						label={ __( 'Edit styles' ) }
						onClick={ async () => await openGlobalStyles() }
					/>
```

Reply:
Then, above:

```jsx
					<SidebarButton
						icon={ seen }
						label={ __( 'Style Book' ) }
						onClick={ async () => {
							await openGlobalStyles();
							setEditorCanvasContainerView( 'style-book' );
						} }
					/>
```

Reply:
Seems to work, I think 🤣

Reply:
Sure does! 🍻